addons/settings/stringtable.xml

```diff
@@ -406,12 +406,16 @@
             <Chinesesimp>此项设定被客户端给覆写并仅适用于伺服器。</Chinesesimp>
             <Portuguese>Esta configuração é sobrescrita pelos clientes e é apenas aplicada no servidor.</Portuguese>
             <Russian>Этот параметр перезаписан клиентом и применяется исключительно к серверу.</Russian>
-            <Polish>Ta wartość zostałą nadpisana przez klientów i ma zastosowanie tylko na serwerze.</Polish>
+            <Polish>Ta wartość została nadpisana przez klientów i ma zastosowanie tylko na serwerze.</Polish>
             <French>Ce paramètre est écrasé par les clients et ne s'applique qu'au serveur.</French>
             <Turkish>Oyuncular tarafından bu ayarın üstüne yazılmış, ayar sadece sunucuda geçerli.</Turkish>
             <Italian>Questa impostazione è sovrascritta dai client e si applica solo al server.</Italian>
             <Czech>Toto nastavení je přepsáno klienty a platí pouze na serveru.</Czech>
         </Key>
+        <Key ID="STR_cba_settings_overwritten_equal">
+            <English>\nOverwriting value is equal.</English>
+            <Polish>\nNadpisująca wartość jest identyczna.</Polish>
```

Comment:
```suggestion
            <Polish>\nNadpisująca wartość jest identyczna.</Polish>
            <French>\nLa valeur de remplacement est la même.</French>
```

Reply:
We decided to change this string. Will happen soonTM.